### ISSUE TITLE
Feature request : support for multiple fido2 devices 

### ISSUE THREAD

**Issue by GaelC92** [rfjakob/gocryptfs#572]
Use case is mainly as a safety net (backup fido2 device), but could be used for multiple user access.  

I figure this could be based on a k in n shared secret algorithm like shamir secret sharing. 
for a start, it would also require to allow multiple entries in the .conf file.

**Comment by deutschem1** [comment]
yes, the main case is for backup. If I init with a single FIDO2 key and I will lose the key i will lose all my encrypted data? (or do I have additionally a masterkey?, dont tried yet). So I think it is common to add multiple keys.

Maybe (but not really important,eventually a feature request): the option to use Password OR a hardware key...

**Comment by rfjakob** [comment]
Yes you always have the master key for recovery.
